src/lib/core/theming/_palette.scss

```diff
@@ -653,7 +655,7 @@ $mat-light-theme-background: (
   dialog:     white,
   disabled-button: $black-12-opacity,
   raised-button: white,
-  dark-overlay: $black-12-opacity,
+  dark-overlay: $black-6-opacity,
```

Comment:
can we make the name for this focused-button to match disabled-button and raised-button above? dark-overlay is kind of a misnomer in the dark theme anyways